dotCMS/src/main/java/com/dotmarketing/util/TaskLocatorUtil.java

```diff
@@ -122,168 +123,180 @@ public static void removeFixTask(Class<?> clazz){
+		.add(Task04115LowercaseIdentifierUrls.class)
+		.add(Task04120IncreaseHostColumnOnClusterServerTable.class)
+		.add(Task04200CreateDefaultVanityURL.class)
+		.add(Task04205MigrateVanityURLToContent.class)
+        .add(Task04210CreateDefaultLanguageVariable.class)
+		.add(Task04215MySQLMissingConstraints.class)
+		.add(Task04220RemoveDeleteInactiveClusterServersJob.class)
+		.add(Task04230FixVanityURLInconsistencies.class)
+		.add(Task04300UpdateSystemFolderIdentifier.class)
+		.add(Task04305UpdateWorkflowActionTable.class)
+		.add(Task04310CreateWorkflowRoles.class)
+		.add(Task04315UpdateMultiTreePK.class)
+		.add(Task04320WorkflowActionRemoveNextStepConstraint.class)
+		.add(Task04235RemoveFKFromWorkflowTaskTable.class)
+		.add(Task04330WorkflowTaskAddLanguageIdColumn.class)
+		.add(Task04335CreateSystemWorkflow.class)
+		.add(Task04340TemplateShowOnMenu.class)
+		.add(Task04345AddSystemWorkflowToContentType.class)
+		.add(Task04350AddDefaultWorkflowActionStates.class)
+		.add(Task04355SystemEventAddServerIdColumn.class)
+		.add(Task04360WorkflowSchemeDropUniqueNameConstraint.class)
+		.add(Task04365RelationshipUniqueConstraint.class)
+		.add(Task04370AddVisitorLogger.class)
+		.add(Task04375UpdateColors.class)
+		.add(Task04380AddSubActionToWorkflowActions.class)
+		.add(Task04385UpdateCategoryKey.class)
+		.add(Task04390ShowEditingListingWorkflowActionTable.class)
+		.add(Task05030UpdateSystemContentTypesHost.class)
+		.add(Task05035CreateIndexForQRTZ_EXCL_TRIGGERSTable.class)
+		.add(Task05040LanguageTableIdentityOff.class)
+		.add(Task05050FileAssetContentTypeReadOnlyFileName.class)
+		.add(Task05060CreateApiTokensIssuedTable.class)
+		.add(Task05070AddIdentifierVirtualColumn.class)
+		.add(Task05080RecreateIdentifierIndex.class)
+		.add(Task05150CreateIndicesForContentVersionInfoMSSQL.class)
+		.add(Task05160MultiTreeAddPersonalizationColumnAndChangingPK.class)
+		.add(Task05165CreateContentTypeWorkflowActionMappingTable.class)
+		.add(Task05170DefineFrontEndAndBackEndRoles.class)
+		.add(Task05175AssignDefaultActionsToTheSystemWorkflow.class)
+	    .add(Task05180UpdateFriendlyNameField.class)
+    	.add(Task05190UpdateFormsWidgetCodeField.class)
+		.add(Task05195CreatesDestroyActionAndAssignDestroyDefaultActionsToTheSystemWorkflow.class)
+		.add(Task05200WorkflowTaskUniqueKey.class)
+		.add(Task05210CreateDefaultDotAsset.class)
+		.add(Task05215AddSystemWorkflowToDotAssetContentType.class)
+		.add(Task05220MakeFileAssetContentTypeBinaryFieldIndexedListed.class)
+        .add(Task05225RemoveLoadRecordsToIndex.class)
+        .add(Task05300UpdateIndexNameLength.class)
+        .add(Task05305AddPushPublishFilterColumn.class).build();
+        
+        return ret.stream().sorted(classNameComparator).collect(Collectors.toList());
+
+
 	}
 
+	
+    final static private Comparator<Class<?>> classNameComparator = new Comparator<Class<?>>() {
+        public int compare(Class<?> o1, Class<?> o2) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'o1' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5632525)